### ISSUE TITLE
Accessibility issue: Page should contain a level-one heading on /az_core/mobile_nav_callback/menu_link_content%3Add562382-61e2-44cd-adce-76db09d7af3f

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#82]
## What
An accessibility scan flagged the element `<html>` because page should contain a level-one heading. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/page-has-heading-one?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/fd4b4464-0b32-455f-94ca-1ce9362b862f.png)

To fix this, ensure that the page, or at least one of its frames contains a level-one heading.

Specifically:

Fix all of the following:
-   Page must have a level-one heading

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
